### ISSUE TITLE
Issues fixed in main are not released

### ISSUE THREAD

**Issue by velxundussa** [ParticulaCode/FoundryVTT-GoDice#29]
Hello,

Multiple issues fixed in main have not been released, so updating to the latest version from Foundry leads to a broken state.

For example, 2.0 doesn't make dice blink when they need to be rolled.

I was able to test that a release from a fork works as intended.

Please make a release to make updating easier for everyone :)

**Comment by lowkeyframe** [comment]
In the meantime, I used your fix. Thx so much, work as intended!

**Comment by TheXeno30** [comment]
Where can i find this fix :)

**Comment by lowkeyframe** [comment]
> Where can i find this fix :)

You can go to velxundussa repo where you will find the fork. Here's a copy for faster reference:

https://github.com/velxundussa/FoundryVTT-GoDice

**Comment by TheXeno30** [comment]
Thanks!
I'm on the forge i'm a little unsure as to how I install the module from github :)

**Comment by velxundussa** [comment]
I don't use the forge , but on a normal instanse you can specify a module.json link.

That is the link to use: 
https://github.com/velxundussa/FoundryVTT-GoDice/releases/download/2.2/module.json

**Comment by TheXeno30** [comment]
Thanks it works - just couldn't fiind the manifest file on the Github  - new to that site

**Comment by lowkeyframe** [comment]
> Please be aware that if the official module publishes an update, it won't be grabbed automatically since you installed it from my repo.
> 
> When (if?) this issue get closed, you'll need to uninstall my release before reinstlling the official one.

Totally!! Thx again for opening the issue/forked a solution. I know its not a big feature, but it makes a big difference at my table.

**Comment by TheXeno30** [comment]
Yes, thanks for all the small fixes. It make a lot of difference.

**Comment by TheXeno30** [comment]
> Please be aware that if the official module publishes an update, it won't be grabbed automatically since you installed it from my repo.
> 
> When (if?) this issue get closed, you'll need to uninstall my release before reinstlling the official one.

Dear Velxundussa
Seems like you are the only thing holding this foundry module alive. Is it possible for you to have a look at some of the many issues with the module? A no is fully acceptable, but I thought that someone had to ask :)

**Comment by ceejii** [comment]
There was a [release (2.1)](https://github.com/ParticulaCode/FoundryVTT-GoDice/releases/) made in April, 2025. It fixes the blinking for the requested dice to roll for me. If you installed the forked release from [velxundussa](https://github.com/velxundussa), you should propably uninstall it and install the official release from Particula to be able to get future updates easily.

[velxundussa](https://github.com/velxundussa) Maybe consider closing this issue now?
